src/components/tx-flow/common/TxNonce/index.tsx

```diff
@@ -102,11 +95,23 @@ enum TxNonceFormFieldNames {
   NONCE = 'nonce',
 }
 
+enum ErrorMessages {
+  NONCE_MUST_BE_NUMBER = 'Nonce must be a number',
+  NONCE_TOO_LOW = "Nonce can't be lower than %%nonce%%",
+  NONCE_TOO_HIGH = 'Nonce is too high',
+  NONCE_TOO_FAR = 'Nonce is very far from the current nonce',
```

Comment:
Maybe something like "Nonce is much higher than the current nonce"? "very far" sounds ambiguous imo

Reply:
Maybe! What does the BritGPT say, @iamacook?

Reply:
The suggestion sounds good to me.